src/transports/ipc/bipc.c

```diff
@@ -168,6 +168,11 @@ static void nn_bipc_destroy (struct nn_epbase *self)
 static void nn_bipc_shutdown (struct nn_fsm *self, int src, int type,
     void *srcptr)
 {
+#if !defined NN_HAVE_WINDOWS
```

Comment:
Actually, nevermind: `AF_UNIX` on windows is defined in `winsock2.h`, so we can feature test with `sys/socket.h`.